src/Config/SDK/ComponentProvider/Metrics/AggregationResolverExplicitBucketHistogram.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Config\SDK\ComponentProvider\Metrics;
+
+use OpenTelemetry\Config\SDK\Configuration\ComponentProvider;
+use OpenTelemetry\Config\SDK\Configuration\ComponentProviderRegistry;
+use OpenTelemetry\Config\SDK\Configuration\Context;
+use OpenTelemetry\Config\SDK\Configuration\Validation;
+use OpenTelemetry\SDK\Metrics\DefaultAggregationProviderInterface;
+use OpenTelemetry\SDK\Metrics\DefaultAggregationProviderTrait;
+use Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition;
+
+/**
+ * @implements ComponentProvider<DefaultAggregationProviderInterface>
+ */
+final class AggregationResolverExplicitBucketHistogram implements ComponentProvider
+{
+
+    /**
+     * @param array{
+     *      boundaries: array<int|float>,
+     *      record_min_max: bool,
+     *  } $properties
+     */
+    public function createPlugin(array $properties, Context $context): DefaultAggregationProviderInterface
+    {
+        // TODO Implement proper aggregation providers (default, drop, explicit_bucket_histogram, last_value, sum) to handle advisory
+        return new class() implements DefaultAggregationProviderInterface {
+            use DefaultAggregationProviderTrait;
+        };
+    }
+
+    public function getConfig(ComponentProviderRegistry $registry): ArrayNodeDefinition
+    {
+        $node = new ArrayNodeDefinition('explicit_bucket_histogram');
+        $node
+            ->children()
+                ->arrayNode('boundaries')
+                    ->prototype('scalar')->validate()->always(Validation::ensureNumber())->end()->end()
+                ->end()
+                ->booleanNode('record_min_max')->defaultValue(false)->end()
```

Comment:
> I'm not sure whether we should add not (yet) supported component providers (personally would prefer failing fast).

Failing fast is fine, my only concern was the hacking required on kitchen-sink.yaml so that our tests could parse it. That's not a particularly big concern, though.

Reply:
We could move the dummy implementations into the `/tests` directory and manually register them for tests using `ServiceLoader::register()` (or add them to the root `composer.json` without adding them to any package `composer.json`).

Reply:
moved dummy implementations into `/tests`